Core/ALife.Core/Scenarios/Mazes/MazeScenario.cs

```diff
@@ -140,7 +141,8 @@ public virtual void PlanetSetup()
 
             for(int i = 0; i < numAgents; i++)
             {
-                Agent rag = AgentFactory.CreateAgent("Agent", red, blue, ColorExtensions.GetRandomColor(), 0);
+                Colour randomColour = Colour.GetRandomColour(Planet.World.NumberGen, 255, 255, redMin: 100, greenMin: 100, blueMin: 100);
```

Comment:
There should be a default for this that is fully random, or "StandardRandom" something like that.

Reply:
The default is fully random. I am constraining it here to what you had before. I thought about editing the defaults rather than this, but before I did I wanted to discuss what we wanted to potentially allow and how we want to handle colours going forwards

Reply:
Made it more like how I originally intended in using the Range objects to control (and if no range object is provided for a "channel" it defaults to one that we have in, essentially, a constant file).